README.md

```diff
@@ -11,9 +11,9 @@ MyRadio is part of a suite of upcoming public projects, including:
 - loggerng, our python audio logging and retriving system
 - Bootstrapping scripts for setting up and configuring all the dependencies
 
-Super Quickstart
+Super Quickstart [@]
```

Comment:
Why the @'s?

Reply:
Just a reference point - could remove it it's just how I like to refer to things

Reply:
i thought this, but am just 💩

Reply:
bin pls :)